react-components/src/components/View/Image360HistoricalDetails.tsx

```diff
@@ -72,6 +74,7 @@ export const Image360HistoricalDetails = ({
   useEffect(() => {
     const newMinWidth = revisionDetailsExpanded ? '100%' : '100px';
     setMinWidth(newMinWidth);
+    onExpand!(revisionDetailsExpanded);
```

Comment:
This will fail if onExpand is not set!

You need to check if it exists before running it. If it is undefined than this will throw an error!